project/Common.scala

```diff
@@ -26,8 +26,6 @@ object Common extends AutoPlugin {
 
     licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
 
-    scalaVersion := crossScalaVersions.value.head,
-    crossScalaVersions := Dependencies.ScalaVersions,
```

Comment:
I see now that version is from `++$TRAVIS_SCALA_VERSION` in the travis script?
how does that work if I build locally and don't specify any scala version?